virtual-desktop/src/app/authentication-manager/login/login.component.ts

```diff
@@ -196,6 +216,16 @@ export class LoginComponent implements OnInit {
         this.renewSession();
       }
     });
+
+    this.idleWarnModal.lastSub=this.lastActive.subscribe(()=> {
```

Comment:
added this new subscription so can cancel idleWarnPopup on activity update in another window